types/index.d.ts

```diff
@@ -1303,6 +1303,12 @@ declare namespace RamdaAdjunct {
          */
         toArray<T>(iterable: Iterable<T> | T): any[];
 
+        /**
+         * Converts value to a number.
+         * It returns `NaN` whenever the global `Number` function would return `NaN` or throw an error.
```

Comment:
```suggestion
         * It returns `NaN` whenever the global `Number` function would return `NaN` or throw an error.
```
Let's drop this sentence. People should expect that this function is safe and never throw any error.